src/test/java/bio/terra/app/controller/SnapshotValidationTest.java

```diff
@@ -170,9 +170,9 @@ public void testSnapshotNameInvalid() throws Exception {
         errorModel = expectBadSnapshotCreateRequest(snapshotByAssetRequest);
         checkValidationErrorModel(errorModel, new String[]{"Size", "Pattern"});
 
-        // Make a 64 character string, it should be considered too long by the validation.
-        // Note: a 63 character string, we are okay with
-        String tooLong = StringUtils.repeat("a", 64);
+        // Make a 513 character string, it should be considered too long by the validation.
```

Comment:
So was this previously incorrect? Shouldn't we have been okay with 64 and not with 65? This feels like an odd pattern switch if not.

Reply:
The max length used to be 63 characters, so this was correct